apps/print/frontend/src/components/number_input.tsx

```diff
@@ -0,0 +1,35 @@
+import { safeParseInt } from '@votingworks/types';
+import styled from 'styled-components';
+
+// if we used a number input here, there is default browser behavior that
+// increases and decreases the value on scroll. instead of preventing this
+// with an event handler, we just use a text input
+const StyledInput = styled.input.attrs({ type: 'text' })`
+  width: 3rem;
+`;
+
+export function NumberInput({
```

Comment:
I see we do something similar in VxAdmin: https://github.com/votingworks/vxsuite/blob/ee3863a6b20b621ba0108ff30a448b5092fa1605/apps/admin/frontend/src/screens/tally/manual_tallies_form_screen.tsx#L173-L178

Could move to libs/ui accordingly, but also okay if we feel that that's preemptive, or want to handle separately

Reply:
<img width="1189" height="753" alt="Screenshot 2025-11-10 at 3 24 49 PM" src="https://github.com/user-attachments/assets/e399b86f-f391-46e9-b569-5d006523dcd6" />

`NumberInput` added to libs/ui with a storybook and replaced `TallyInput` in the manual tallies form within VxAdmin ([commit](https://github.com/votingworks/vxsuite/pull/7463/commits/b39ea3c46d2ec971632350f37ef624fe3c026d85))